charts/workflow/values.yaml

```diff
@@ -51,6 +51,16 @@ global:
   host_port: 5555
   # Prefix for the imagepull secret created when using private registry
   secret_prefix: "private-registry"
+  # The public resolvable hostname to build your cluster with.
+  #
+  # This will be the hostname that is used to build endpoints such as "deis.$HOSTNAME"
+  hostname: ""
```

Comment:
This is analogous to the router chart's `platform_domain` setting, yes?

Can I propose that `platform_domain` is a better name than `hostname`? And to both 1. avoid confusion with the setting of the same name under `router` and 2. move this configuration closer to the point of use, this belongs under `controller` settings instead of `global` (because, afaik, the controller chart, which will have to honor this new setting won't be able to "look up" to the "parent" chart's global settings).